.github/workflows/embedding-sdk.yml

```diff
@@ -140,18 +141,20 @@ jobs:
         run: exit 1
 
   # See `Component e2e tests` in `enterprise/frontend/src/embedding-sdk-package/dev.md`
-  sdk-e2e-tests:
```

Comment:
Unfortunate naming that was causing a lot of confusion, until I really tried to understand what each of these jobs were doing.